phase1/gce/configure-vm-kubeadm.sh

```diff
@@ -31,13 +31,7 @@ else
 fi
 
 if [[ "${KUBEADM_VERSION}" != "${KUBELET_VERSION}" ]]; then
-  if [[ "${KUBEADM_VERSION}" == stable ]]; then
-    # Cannot install packages as they will update the 
-    # kubelet configuration to a version that does not match
-    # the installed kubelet
-    echo "Kubeadm version of 'stable' is not supported with kubelet version that is not also 'stable'."
-    exit 1
-  elif [[ "${KUBEADM_VERSION}" == "gs://"* ]]; then
+  if [[ "${KUBEADM_VERSION}" == "gs://"* ]]; then
```

Comment:
Can we add a comment explaining why we would never support stable kubeadm with kubelet gs version? I think it will help reduce confusion for future readers.